App.js

```diff
@@ -1,26 +1,29 @@
 import React from 'react'
-import styled from 'styled-components/native'
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+// Components
+import StartScreen from './components/StartScreen';
+import TipScreen from './components/TipScreen';
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+// Navigation
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import 'react-native-gesture-handler';
```

Comment:
Thank you dividing dependencies by usage :) Nevertheless, those dependencies should be moved above local ones